newsfragments/743.added.md

```diff
@@ -0,0 +1,3 @@
+Introducing Element Admin, a user-friendly interface to manage your ESS deployment.
+
+<img width="1625" height="961" alt="Element Admin" src="https://github.com/user-attachments/assets/ee37deb6-d7e4-422d-8026-af948fcb251d" />
```

Comment:
Is this really going to work given this file is markdown and is going to end up in GH releases + the changelog in chart + elsewhere for Pro